packages/venia-ui/lib/components/CreateAccountPage/createAccountPage.js

```diff
@@ -1,32 +1,62 @@
 import React from 'react';
 import { shape, string } from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+
 import { useCreateAccountPage } from '@magento/peregrine/lib/talons/CreateAccountPage/useCreateAccountPage';
+import { useStyle } from '@magento/venia-ui/lib/classify';
+import CreateAccount from '@magento/venia-ui/lib/components/CreateAccount';
+import { StoreTitle } from '@magento/venia-ui/lib/components/Head';
 
-import CreateAccountForm from '../CreateAccount';
-import { useStyle } from '../../classify';
 import defaultClasses from './createAccountPage.css';
 
 const CreateAccountPage = props => {
-    const talonProps = useCreateAccountPage();
-    const { initialValues, handleCreateAccount } = talonProps;
+    const { signedInRedirectUrl, signInPageUrl } = props;
     const classes = useStyle(defaultClasses, props.classes);
+    const { handleOnCancel, initialValues } = useCreateAccountPage({
+        signedInRedirectUrl,
+        signInPageUrl
+    });
```

Comment:
Some new patterns have emerged that I think could be applicable here. I know we do an awful job at documenting outside of the core team, so consider these optional but valuable improvements.

1. Start passing raw props to talons, ie. `useCreateAccountPage(props)`. Less important here since you're passing all the props anyway, but gives more power to wrapped talon targets to override original props. 
2. Prop Objects. Instead of the talon returning individual props that get composed in the component, return a prop object that we can spread onto components.

```js
// in the talon
const createAccountProps = {
    initialValues,
    isCancelButtonHidden: false,
    onCancel: handleOnCancel
};

// component
const { createAccountProps } = talonProps;
return <CreateAccount {...createAccountProps} />
```